app/models/user.rb

```diff
@@ -89,6 +89,10 @@ def registration_facility_id
 
   alias facility registration_facility
 
+  def facility_in_facility_group?(facility_id)
```

Comment:
The naming suggestions make sense, fixed. 👍🏽

As for the validations themselves, this is necessary so that the user cannot assign a `registration_facility` or an `assigned_facility` for a patient to a facility group that is outside their purview of access – this would be a data breach of a kind.

If the intention was to allow the user to assign a patient to any facility in the system, then it'd make sense to not have this validation, but since the validation is scoped to their facility group, it should be considered an invalid attempt from an API perspective (even though it's also disallowed from the UI in the app).

Does that seem fair?

Reply:
> If the intention was to allow the user to assign a patient to any facility in the system, then it'd make sense to not have this validation, but since the validation is scoped to their facility group, it should be considered an invalid attempt from an API perspective (even though it's also disallowed from the UI in the app).
> 
> Does that seem fair?

Yup, that makes sense.